frontend/src/ui/TransitionTable.tsx

```diff
@@ -72,6 +72,10 @@ const TransitionTable = ({
           title: 'Average annual salary',
           field: 'annualSalary',
           type: 'currency',
+          render: ({ annualSalary }) =>
+            `$${Math.floor(annualSalary)
```

Comment:
Another way to round a number to fixed number of decimales is [String.toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed). `annualSalary.toFixed()` returns a whole number string